openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -30,24 +30,33 @@
     from pydantic import validator  # type: ignore[assignment]
 
 
-class AlchemicalSettings(SettingsBaseModel):
+class LambdaSettings(SettingsBaseModel):
     class Config:
         extra = 'ignore'
         arbitrary_types_allowed = True
 
-    """Settings for the alchemical protocol
-
-    This describes the lambda schedule and the creation of the
-    hybrid system.
+    """Settings for the lambda protocol
+    This describes the lambda schedule.
```

Comment:
```suggestion
    Lambda schedule settings.
    
    Settings controlling the lambda schedule, these include the switching function type, and the number of windows.
```
Or something like that.